services/actions/notifier_helper.go

```diff
@@ -144,6 +146,10 @@ func notify(ctx context.Context, input *notifyInput) error {
 		return fmt.Errorf("gitRepo.GetCommit: %w", err)
 	}
 
+	if skipped := skipWorkflowsForCommit(input, commit); skipped {
```

Comment:
```suggestion
	if skipWorkflowsForCommit(input, commit) {
```

Why a variable .....